src/sqlancer/questdb/test/QuestDBQueryPartitioningBase.java

```diff
@@ -66,12 +67,15 @@ protected ExpressionGenerator<Node<QuestDBExpression>> getGen() {
     @Override
     public void check() throws SQLException {
         s = state.getSchema();
-        targetTables = s.getRandomTableNonEmptyTables();
-        gen = new QuestDBExpressionGenerator(state).setColumns(targetTables.getColumns());
+        // Only return one table instead of multiple tables, which is regarded as illegal by QuestDB
+        // e.g. "SELECT * FROM t0, t1;"
+        targetTable = s.getRandomTable(t -> !t.isView());
```

Comment:
I think we can also consider views in the `SELECT` statement, so perhaps we can just select any random table?

Reply:
OK, then just `targetTable = s.getRandomTable();` will be fine.